source/developer/mobile-developer-setup.rst

```diff
@@ -117,10 +189,122 @@ Android (Device)
 
 8. The installed APK may not be opened automatically. You may need to manually open the Mattermost app on your device.
 
+Start Developing and Building
+------------------------------------
+
+In order to develop and build the Mattermost mobile apps you'll need to get a copy of the source code. Also, it is best if you fork our mattermost-mobile repo since it is most likely that you'll want to contribute back your work.
```

Comment:
can we change the phrasing on "Also, it is best if you fork our mattermost-mobile repo since it is most likely that you'll want to contribute back your work."

maybe something like the following, or feel free to rewrite to something else: 

"Forking the mattermost-mobile repository will also make it easy to contribute your work back to the project in the future."